mergify_engine/actions/update.py

```diff
@@ -32,10 +38,35 @@ class UpdateAction(actions.Action):
         | actions.ActionFlag.ALLOW_ON_CONFIGURATION_CHANGED
         | actions.ActionFlag.DISALLOW_RERUN_ON_OTHER_RULES
     )
-    validator: typing.ClassVar[typing.Dict[typing.Any, typing.Any]] = {}
+    validator = {
+        voluptuous.Required("bot_account", default=None): types.Jinja2WithNone,
+    }
+
+    async def run(
+        self, ctxt: context.Context, rule: rules.EvaluatedRule
+    ) -> check_api.Result:
+        try:
+            bot_account = await action_utils.render_bot_account(
+                ctxt,
+                self.config["bot_account"],
+                option_name="bot_account",
+                required_feature=subscription.Features.BOT_ACCOUNT,
+                missing_feature_message="Rebase with `update_bot_account` set is unavailable",
+            )
+        except action_utils.RenderBotAccountFailure as e:
+            return check_api.Result(e.status, e.title, e.reason)
+
+        github_user: typing.Optional[user_tokens.UserTokensUser] = None
+        if bot_account:
+            tokens = await ctxt.repository.installation.get_user_tokens()
+            github_user = tokens.get_token_for(bot_account)
+            if not github_user:
+                return check_api.Result(
+                    check_api.Conclusion.FAILURE,
+                    f"Unable to comment: user `{bot_account}` is unknown. ",
```

Comment:
Shouldn't it be
```suggestion
                    f"Unable to update: user `{bot_account}` is unknown. ",
```
?